docs/exporter/otlp/otlp.rst

```diff
@@ -1,12 +1,42 @@
 OpenTelemetry OTLP Exporters
 ============================
-
 .. automodule:: opentelemetry.exporter.otlp
     :members:
     :undoc-members:
     :show-inheritance:
 
+opentelemetry.exporter.otlp.proto.http
+---------------------------------------
+
+.. automodule:: opentelemetry.exporter.otlp.proto.http
+    :members:
+    :undoc-members:
+    :show-inheritance:
+
+.. automodule:: opentelemetry.exporter.otlp.proto.http.trace_exporter
+    :members:
+    :undoc-members:
+    :show-inheritance:
+
+.. automodule:: opentelemetry.exporter.otlp.proto.http.metric_exporter
+    :members:
+    :undoc-members:
+    :show-inheritance:
+
```

Comment:
I think it makes sense to have _log_exporter too

Reply:
I don't see an issue with it seeing as we already have [logs](https://github.com/open-telemetry/opentelemetry-python/blob/main/docs/api/_logs.rst) api in docs even though it is under experimental.